libdnf/repo/Repo.cpp

```diff
@@ -549,6 +549,8 @@ std::unique_ptr<LrHandle> Repo::Impl::lrHandleInitRemote(const char *destdir)
         handleSetOpt(h.get(), LRO_URLS, urls);
     }
 
+    handleSetOpt(h.get(), LRO_SSLVERIFYSTATUS, conf->sslverifystatus().getValue() ? 1L : 0L);
```

Comment:
Similar code needs to be added to the "context" part of libdnf. - into `dnf_repo_set_keyfile_data` function in "dnf-repo.cpp".

Reply:
I have moved the option to the new `setHandle` template and added it to `dnf_repo_setup` for the "context" part, this way the option is together with the rest of ssl options for both libdnf code paths.